bookbotproject/conversationtree/views.py

```diff
@@ -90,29 +93,61 @@ def add_new_user(request, _name, _userid, _chatid):
 
 
 @csrf_exempt
+def add_comment (  request, _title, _userid, _comment ):
+    try:
+        commentbook = Book.objects.get(title=_title)
+    except:
+        Book.objects.create(title=_title)
+        commentbook = Book.objects.get(title=_title)
+    _user = TelegramUser.objects.get(userid=_userid)
+    newComment = Comment.objects.create( user=_user,comment=_comment, book = commentbook)
+    return HttpResponse(status=200)
+
 def get_response(request, _message, _chatid):
     """ waits until a response from wit ai, it may take so much time !!!!"""
-
+    #print('wow')
```

Comment:
we can uncomment this with a more explanatory sentence or we can delete this line altogether